app/src/main/java/com/kickstarter/services/ApiClientV2.java

```diff
@@ -279,8 +279,8 @@ public ApiClientV2(final @NonNull ApiServiceV2 service, final @NonNull Gson gson
 
   @Override
   public @NonNull Observable<Update> fetchUpdate(final @NonNull Update update) {
-    final String projectParam = ObjectUtils.toString(update.projectId());
-    final String updateParam = ObjectUtils.toString(update.id());
+    final String projectParam = AnyExtKt.numToString(update.projectId());
```

Comment:
For java I see it somewhat useful as `Long.toString(update.projectId())` might throw exceptions and the extensions validates the types, but we need to add some tests for the method